common/defs/blockchain_link.json

```
@@ -277,9 +277,9 @@
     "type": "blockbook",
     "url": ["https://pol1.trezor.io", "https://pol2.trezor.io", "https://pol3.trezor.io", "https://pol4.trezor.io"]
   },
-  "eth:tHOL:17000": {
+  "eth:tHOOD:17000": {
```

```suggestion
  "eth:tHOOD:560048": {
```

https://chainid.network/chain/560048/